src/main/java/com/suchocki/parkingmeter/propertyeditor/DatePropertyEditor.java

```diff
@@ -0,0 +1,31 @@
+package com.suchocki.parkingmeter.propertyeditor;
+
+import java.beans.PropertyEditorSupport;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class DatePropertyEditor extends PropertyEditorSupport {
+
+	SimpleDateFormat dateFormat;
```

Comment:
1. dlaczego to ma dostęp pakietowy?
2. Czy SDF może być bezpiecznie użyte w wielu wątkach?

Reply:
1. Przez pomyłkę - zapomniałem dopisać _private_.
2. Teoretycznie może. Przy odpowiedniej synchronizacji/użyciu z `ThreadLocal`, ale nie jest to tutaj zaimplementowane. Lepiej byłoby użyć innej klasy (np. `LocalDate`)